doc/rados/operations/crush-map.rst

```diff
@@ -542,22 +532,24 @@ The following example removes the ``rack12`` bucket from the hierarchy:
 Creating a compat weight set
 ----------------------------
 
-.. note: This step is normally done automatically by the ``balancer``
-   module when enabled.
+.. note:: Normally this action is done automatically by the ``balancer`` module
```

Comment:
automatically if needed by the

Few people should be using the crush-compat mode with the most recent releases.

Reply:
Accepted.